### ISSUE TITLE
Mac OS X only-32 and only-64 not included in export template (only fat 32+64 is)

### ISSUE THREAD

**Issue by IBwWG** [godotengine/godot#7290]
**Operating system or device - Godot version:**
win7x64, godot 2.1.1

**Issue description** (what happened, and what was expected):
Try to export project to 64-bit or 32-bit.  Alert says error exporting.  Command window says,

```
ERROR: EditorExportPlatformOSX::export_project: Requested template binary 'godot
_osx_release.64' not found. It might be missing from your template archive.
   At: platform\osx\export\export.cpp:424
ERROR: ProjectExportDialog::export_platform: Exporting failed!
   At: tools\editor\project_export.cpp:555
```

**Steps to reproduce:**
Install export templates.  Export.  Mac OSX.  Choose non-fat option for Bits Mode.  Hit Export.

**Link to minimal example project** (optional but very welcome):

**Comment by lukaskotik** [comment]
I confirm the same issue in Godot 2.1.4.

**Comment by BastiaanOlij** [comment]
Yes, only the fat binary is there. There is currently some discussion going on whether it makes more sense to only supply the 64bit version and let go of 32bit as Apple is starting to drop support for it anyway.

It is questionable whether it makes sense to include both 32bit and 64bit builds along side the fat build as it just makes the download bigger with very little added benefit.

I think the option to switch was more there for those who do custom builds. I could be wrong there as this all played out long before I got involved into things :)

**Comment by Noshyaar** [comment]
First of all thank you for your report and sorry for the delay.

We released Godot 3.0 in January 2018 after 18 months of work, fixing many old issues either directly, or by obsoleting/replacing the features they were referring to.

We still have hundreds of issues whose relevance/reproducibility needs to be checked against the [current stable version](https://godotengine.org/download), and that's where you can help us.
**Could you check if the issue that you described initially is still relevant/reproducible in Godot 3.0 or any newer version, and comment about it here?**

For bug reports, please also make sure that the issue contains detailed steps to reproduce the bug and, if possible, a zipped project that can be used to reproduce it right away. This greatly speeds up debugging and bugfixing tasks for our contributors.

Our Bugsquad will review this issue more in-depth in 15 days, and potentially close it if its relevance could not be confirmed.

Thanks in advance.

*Note: This message is being copy-pasted to many "stale" issues (90+ days without activity). It might happen that it is not meaningful for this specific issue or appears oblivious of the issue's context, if so please comment to notify the Bugsquad about it.*

**Comment by Calinou** [comment]
Since 32-bit macOS is no longer supported as of Godot 3.0 (only 64-bit macOS is supported now), this issue can now be closed.
